doc/source/whatsnew/v1.5.0.rst

```diff
@@ -481,7 +481,7 @@ Timedelta
 
 Time Zones
 ^^^^^^^^^^
--
+- Bug in :class:`Timestamp` constructor raising when passed a ``ZoneInfo`` tzinfo object (:issue:`46425`)
```

Comment:
Could it be more of a enhancement that all `Timstamp/DTI/etc` can all accept ZoneInfo objects now?